SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -200,6 +200,61 @@ bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream)
 	return true;
 }
 
+bool CreateWorkerAuthorityAssignmentEntities(Worker_SnapshotOutputStream* OutputStream)
+{
+	// For each worker type we need to ensure that they are authoritative over some entity component to ensure
+	// that they receive updates for the GSM if the worker is yet to receive authority over a component
+	// on an actor created during gameplay.
+
+	// Long-term we may be able to replace this with system entities.
+	const float CHUNK_SIZE = 5.0f; // in SpatialOS coordinates.
+
+	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
+	int32 WorkerAuthorityAssignmentEntityIdCounter = SpatialGDKSettings->bGeneratePlaceholderEntitiesInSnapshot ? SpatialConstants::PLACEHOLDER_ENTITY_ID_LAST + 1 : SpatialConstants::PLACEHOLDER_ENTITY_ID_FIRST;
```

Comment:
I feel like as these entity ids might not be genuine anymore, we should just remove them entirely. Can we just pass the next available entity id, and increment as we go? The only static entity ids we need are the gsm and player spawner.

Reply:
Yep @alastairdglennie requested the same change, will do that now:)